src/index.html

```diff
@@ -8,9 +8,40 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+     rel="stylesheet"
+     href="./style.css"
+     >
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     data-qa="big"
+     class="common_form big_form"
+     action="#"
+     method="post"
+     >
```

Comment:
attributes must have 2 space indents
```suggestion
    <form
      data-qa="big"
      class="common_form big_form"
      action="#"
      method="post"
    >
```